src/components/apptCard/ApptCard.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+export const ApptCard = () => {
+  return (
+    <div style={{border: "1px solid black"}}>
```

Comment:
Can we put all the styles into an `index.css`? This will make the conversion to the Author easier.

Reply:
This is temporary. I'll remove.